packages/near-fast-auth-signer/.gitignore

```diff
@@ -25,3 +25,6 @@ yarn-error.log*
 
```

Comment:
Not sure if this is ok, but when there is a error message, the `Connect Wallet` will be almost touching the lower border of the modal. 

<img width="394" alt="Screenshot 2024-01-25 at 3 25 53 PM" src="https://github.com/near/fast-auth-signer/assets/22358004/f8d3695f-4b3f-4e27-b7b4-705049701171">